documentation/modules/ref-address-space-example-exposing-endpoints.adoc

```diff
@@ -70,7 +69,9 @@ spec:
      routeHost: messaging.example.com <5>
 ----
 <1> (Required) The name of the endpoint. The name specified affects the name of the {KubePlatform} service to be created as well as the name of the endpoint in the status section of the `AddressSpace`.
-<2> (Required) The service names that must be configured for the `AddressSpace`.  The service can either be `messaging`, `console`, or `mqtt`. However, the `mqtt` service is supported for the `standard` address space type only.
-<3> (Required) The name of the port to be exposed. With the `route` type, only a single TLS-enabled port can be specified. For the `messaging` service, either `amqps` and `https` can be specified. For the `console` service, only `https` can be specified.
-<4> (Required) The TLS termination policy to be used for the OpenShift route. For the `messaging` service, the `amqps` port requires `passthrough` to be used, whereas `https` (websockets) also allows `reencrypt`. For the `console` service, `reencrypt` can be used.
-<5> (Optional) The hostname to use for the created route.
+<2> (Required) The service names that must be configured for the `AddressSpace`.  Valid values are `messaging`, `console`, or `mqtt`. However, the `mqtt` service is supported for the `standard` address space type only.
+<3> (Required) The name of the port to be exposed. With the `route` type, only a single TLS-enabled port can be specified. For the `messaging` service, `amqps` or `https` can be specified. For the `console` service, only `https` can be specified.
+<4> (Required) The TLS termination policy to be used for the {KubePlatform} route. For the `messaging` service, the `amqps` port requires `passthrough` to be used, whereas `https` (websockets) also allows `reencrypt`. For the `console` service, `reencrypt` can be used.
```

Comment:
Again, routes are Openshift only so it does not make sense to talk about a Kubernetes route - is this in a block which is excluded for Kubernetes